client/blocks/image-editor/test/image-editor-toolbar.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'ImageEditorToolbar', () => {
+	useFakeDom();
+
+	let ImageEditorToolbar,
+		defaultProps,
+		wrapper;
+
+	useSandbox( ( sandbox ) => {
+		defaultProps = {
+			translate: sandbox.stub().returns( 'Hajimemashite!' ),
+			onShowNotice: sandbox.spy()
+		};
+	} );
+
+	before( () => {
+		ImageEditorToolbar = require( '../image-editor-toolbar' ).ImageEditorToolbar;
+	} );
+
+	beforeEach( () => {
+		wrapper = shallow( <ImageEditorToolbar { ...defaultProps } /> );
+	} );
+
+	it( 'should insert `is-disabled` class when isGreaterThanMinimumDimensions === `false`', () => {
```

Comment:
Can we think of rephrasing these test cases to describe more of the intent here?